### PR TITLE
En us

### PR DESCRIPTION
**Pull request by archae0pteryx**
Heres a chunk for ya.

### CODE REVIEW

File: 01-1st-glance/README.md [hunk 1]
```diff
@@ -1,56 +1,52 @@
...
+Since 1.0 was released on May 15, the emergence of a large number of outstanding items (you can find on github search Rust), large companies are increasingly active participation Rust application development, as well as the open source community feedback.
 
-自 15 年 5 月 1.0 发布以来，涌现了大量优秀的项目（可以 github 上搜索 Rust 查找），大公司也逐渐积极参与 Rust 的应用开发，以及回馈开源社区。
+The book (RustPrimer) aims to provide a proper Chinese Rustaceans beginners, the latest, to understand Chinese tutorial. This book would have been perfect follow up, never stops.
 
-本书（RustPrimer）旨在为中文 Rustaceans 初学者提供一个正确、最新、易懂的中文教程。本书会一直完善跟进，永不停歇。
+This book is the result of the entire Chinese community Gongtongnuli of Rust. Wherein, Rustacean involved in the book writing and revision are (in alphabetical order):
 
-本书是整个 Rust 中文社区共同努力的结果。其中，参与本书书写及校订的 Rustacean 有（排名不分先后）：
+daogangtang (Mike Cat)
```
Comment (MarisaKirisame) [inline 01-1st-glance/README.md]: Do not translate the name.
Also where's me(

Comment (archae0pteryx) [inline 01-1st-glance/README.md]: You'll have to let me know when I do. I wish I could speak chinese...

Comment (daogangtang) [inline 01-1st-glance/README.md]: enen, just adjust.

Comment (daogangtang) [inline 01-1st-glance/README.md]: thanks for your hard work.

**Comment by MarisaKirisame** [comment]
I'd say you could leave the important chapter untranslated, and I will do it.
Also what version are you basing on? It seems quite old...

File: 01-1st-glance/README.md (hunk 1 above)
Comment (MarisaKirisame) [inline 01-1st-glance/README.md]: I dont think name is something we could, like, adjust later, it should be something always done right.
But beyond that I whole heartedly agree with using google translate.

**Pull request closed without merging**